lab_1_classify_by_unigrams/start.py

```diff
@@ -1,20 +1,48 @@
-"""
-Language detection starter
-"""
+"""Language detection starter"""
+from lab_1_classify_by_unigrams.main import create_language_profile, detect_language_advanced, \
```

Comment:
1. try to add empty line after the doc string
2. Revert doc string to the previous version (multiline)
3. try to use `(` and `)` instead of `\`

Reply:
I did. Still it doesn't work

Reply:
I did. Still it doesn't work